x11-packages/kf6-kcmutils/build.sh

```diff
@@ -0,0 +1,50 @@
+TERMUX_PKG_HOMEPAGE="https://invent.kde.org/frameworks/kcmutils"
+TERMUX_PKG_DESCRIPTION="Utilities for interacting with KCModules"
+TERMUX_PKG_LICENSE="LGPL-2.0-only, LGPL-3.0-only"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="6.21.0"
+_KF6_MINOR_VERSION="${TERMUX_PKG_VERSION%.*}"
+TERMUX_PKG_SRCURL="https://download.kde.org/stable/frameworks/${_KF6_MINOR_VERSION}/kcmutils-${TERMUX_PKG_VERSION}.tar.xz"
+TERMUX_PKG_SHA256="9c1808153d9c16aaa08e68b6353ce5c9af67b2e24b9a0cd7587c46a4a81e8edc"
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_DEPENDS="kf6-kconfig, kf6-kconfigwidgets, kf6-kcoreaddons, kf6-kguiaddons, kf6-ki18n, kf6-kio, kf6-kirigami, kf6-kitemviews, kf6-kservice, kf6-kwidgetsaddons, kf6-kxmlgui, libc++, qt6-qtbase, qt6-qtdeclarative"
+TERMUX_PKG_BUILD_DEPENDS="extra-cmake-modules (>= ${_KF6_MINOR_VERSION}), qt6-qttools"
```

Comment:
In this case there is a second instance but it's probably still preferable to not introduce a global helper variable to the build.
```suggestion
TERMUX_PKG_BUILD_DEPENDS="extra-cmake-modules (>= ${TERMUX_PKG_VERSION%.*}), qt6-qttools"
```